webapp-django/staticfiles/.eslintrc

```diff
@@ -0,0 +1,15 @@
+rules:
+  linebreak-style:
+    - 2
+    - unix
+  max-len: "off" # Use prettier default line length
+env:
+  browser: true
+  es6: true
+parserOptions:
+  ecmaVersion: 7
+  sourceType: module
+extends:
+  - eslint:recommended
+  - plugin:prettier/recommended
+  - plugin:react/recommended
```

Comment:
I don't know offhand what this file means, but we do have a very short list of JavaScript code conventions documented here:

https://socorro.readthedocs.io/en/latest/contributing.html#javascript-code-conventions

Does this file encode those conventions or should we update our conventions?

Reply:
From what I remember, we opted for 2-space indentation for https://github.com/mozilla-services/socorro/blob/master/.editorconfig to reduce the amount of changes.

Reply:
That file has us at a 4-space indent.

It's entirely possible the conventions didn't match the code. I don't do much in JS-land, so I don't know what the deal is.

Seems like we have mis-matched things. We should update all the things to be either 2-space or 4-space. We could throw that in a bug to do later.

Reply:
I think prettier defaults to tab space of 2.

Reply:
2 spaces is preferred, especially for React code, since with JSX you end up with some pretty deep nesting and the 2-space indents really help.

Given that, I think keeping this separate from the rest of the site's styles is appropriate, at least until we decide if the cupcake is a good idea and should be exported to the rest of the site.